cornerstone_widget/cs_widget.py

```diff
@@ -149,32 +155,48 @@ def __init__(self, buttons_per_row=3,
                                      icon="play",
                                      button_style="success"
                                      )
+
         # We use the refresh button as a "start" button to
         # show the first image and then replace the on_click
         # handler after the first click
 
         def _first_click(button):
-            button.description = "Reset"
-            button.icon = "refresh"
-            button.button_style = ""
-            self._refresh_image()
+            # type: (widgets.Button) -> None
+            button.disabled = True
```

Comment:
maybe move this to L178. It is confusing to first always disable the button but then in one case re-enable it again. Took me a moment to find that re-enabling.

Reply:
I decided the debouncing behavior is useful enough and confusing enough that we should just encapsulate it. There are plenty of other buttons we can/should use it for. For the MultipleChoiceWidget and the like we need to decide what the appropriate response is if the callback fails